packages/firebase_storage/firebase_storage/example/ios/Podfile

```diff
@@ -28,6 +28,8 @@ require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelpe
 flutter_ios_podfile_setup
 
 target 'Runner' do
+  use_frameworks!
+  use_modular_headers!
```

Comment:
I know use_frameworks! is a strict requirement of firebase-ios-sdk these days, but I'm unfamiliar with `use_modular_headers` being required. Is modular headers required now as well?

Reply:
I've just tested without it and I guess it isn't needed. I was under the impression it was needed along with `use_frameworks!` 👍